test/e2e-api/members/create-stripe-checkout-session.test.js

```diff
@@ -0,0 +1,45 @@
+const {agentProvider, mockManager, fixtureManager, matchers} = require('../../utils/e2e-framework');
+
+let membersAgent, adminAgent, membersService;
+
+describe('Create Stripe Checkout Session', function () {
```

Comment:
Can we add a happy-path test here as well?

Reply:
I'll take a look at that now - we'll have to mock the Stripe API but should be easy enough